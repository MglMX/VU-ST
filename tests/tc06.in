2
h
i
u
exit
