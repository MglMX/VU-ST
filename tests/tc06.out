Choose category to play or type exit to leave
(1) Cars
(2) Animals
Type category number:==============
Category: Animals
Letters: []
Lives left: 6
Word to guess: _ _ _ _ _
==============
Category: Animals
Letters: ['h']
Lives left: 6
Word to guess: h _ _ _ _
==============
Category: Animals
Letters: ['h', 'i']
Lives left: 6
Word to guess: h i _ _ _
You missed!
==============
Category: Animals
Letters: ['h', 'i', 'u']
Lives left: 5
Word to guess: h i _ _ _
Thank you for playing!
